android/kotlin/v2/AmpliApp/app/src/main/java/com/amplitude/ampli/Ampli.kt

```diff
@@ -443,25 +444,25 @@ open class Ampli {
     private var disabled: Boolean = false
 
     @android.annotation.SuppressLint("StaticFieldLeak")
-    private var _client: Amplitude? = null
```

Comment:
Not sure that we should make this externally settable. Thinking we should make it read only. WDYT? cc @qingzhuozhen

Reply:
I believe this can be marked as private set with public get